spec/lib/tasks/forms.rake_spec.rb

```diff
@@ -78,6 +79,48 @@
         end
       end
 
+      context "with a single form" do
+        let(:form) { build :form, id: 10, organisation_id: form_organisation&.id }
+        let(:forms) { [form] }
+
+        let(:update_organisation_request) do
+          ActiveResource::Request.new(
+            :put,
+            "/api/v1/forms/#{form.id}",
+            hash_including(organisation_id: group.organisation_id),
+            put_headers,
+          )
+        end
+
+        before do
+          task.invoke(form.id, group.external_id)
+        end
+
+        context "and not in an organisation" do
+          let(:form_organisation) { nil }
+
+          it "does not change the group organisation" do
+            expect(ActiveResource::HttpMock.requests).not_to include update_organisation_request
+          end
+        end
+
+        context "and in the same organisation as the group" do
+          let(:form_organisation) { group.organisation }
+
+          it "does not change the group organisation" do
+            expect(ActiveResource::HttpMock.requests).not_to include update_organisation_request
+          end
+        end
+
+        context "and in a different organisation to the group" do
+          let(:form_organisation) { create :organisation, slug: "other-org" }
+
+          it "changes the group organisation" do
```

Comment:
```suggestion
          it "changes the form's organisation" do
```

Reply:
Ah nice catch, thanks!

Reply:
Fixed in https://github.com/alphagov/forms-admin/compare/378d2171551d4ba942f0b442a73c89f3a578928a..341b71b436924a596d1f21a5588069dc2772b18b